src/test/java/org/example/tests/api/rest/wrapper/user/post/GETPostsTest.java

```diff
@@ -15,12 +14,12 @@
 public class GETPostsTest extends ApiBaseClass {
     @Test
     public void getPostList() {
-        Response response = restWrapper.sendRequest(HttpMethod.GET, "/post", "", "");
 
-        logResponse(response);
+        PostsRequests request = new PostsRequests(restWrapper);
```

Comment:
You don't need to create a request object specific to each test. Please create it in a before class method.